### PR TITLE
NikonDataReader example

### LINKED ISSUE #168: How-to load data and create an AcquisitionData: NikonDataReader
Add how-to notebooks showing how to use the NikonDataReader

### PR DESCRIPTION
**Pull request by hrobarts**

### CODE REVIEW

File: how-to/1_Read_and_visualise/NikonDataReader.ipynb
Comment (lauramurgatroyd) [inline how-to/1_Read_and_visualise/NikonDataReader.ipynb:1]: Not sure why copyright years are 2021-22?

**Review by lauramurgatroyd — APPROVED** [review]
Hi, I think it looks great.

Not sure where we got to in terms of whether the notebook needs to be rendered here or not, but in terms of the content, happy to approve.

Some possible suggestions for changes:
- Could clarify that normalise=True normalises loaded projections by detector white level (I_0)

- I was thinking about the features of the Nikon Reader and if everything is covered:
- fliplr isn't covered but I am not sure how its useful
- the `get_geometry` method isn't covered and how it can be used to retrieve geometry from a Nikon file without reading the data. Do you think its worth covering this capability? It's something I have made use of before

**Comment by M-A-Demir** [comment]
Hi Hannah, the how-to is very clear and easy to follow from my perspective.

**Review by M-A-Demir — APPROVED** [review]

**Comment by lauramurgatroyd** [comment]
get_geometry addition looks great!

**Review by M-A-Demir — APPROVED** [review]

**Review by M-A-Demir — APPROVED** [review]

**Comment by hrobarts** [comment]
Thank you for the reviews @lauramurgatroyd and @M-A-Demir 
I added examples for get_geometry and fliplr, explained the Normalise=False and fixed the license date

**Pull request merged**